package.json

```diff
@@ -105,8 +95,8 @@
     "test": "TZ=utc jest src/* && npm run test:e2e",
     "test:server": "jest test/server/*",
     "test:e2e": "./scripts/run_e2e_tests.sh",
-    "build:server": "./node_modules/.bin/babel -D ./src -d ./build --ignore widget.js && cp src/templates/widget.js build/templates/",
-    "build:langs": "rm -rf build/messages && ./node_modules/.bin/babel src > /dev/null && ./node_modules/.bin/babel-node scripts/translate.js",
+    "build:server": "babel -D ./src -d ./build --ignore widget.js && cp src/templates/widget.js build/templates/",
+    "build:langs": "rm -rf build/messages && babel src > /dev/null && babel-node scripts/translate.js",
```

Comment:
is this really necessary? I'm worried that this may mess up with people's environment. It's better to make sure that they use the version specified in this package.json.